admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyModal.vue

```diff
@@ -0,0 +1,121 @@
+<!--**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *-->
+<template>
+  <div class="modal fade" id="ps-modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">{{trans('Customize symbol and format')}} - {{ editedLanguage.name }}</h5>
+          <button type="button" class="close" data-dismiss="modal">
+            <i class="material-icons">close</i>
+          </button>
+        </div>
+
+        <div class="modal-body">
+          <currency-format-form v-if="showCustomizer"></currency-format-form>
+        </div>
+        <div class="modal-footer">
+          <PSButton @click="saveCustomCurrency" class="btn-lg" primary data-dismiss="modal">{{trans('Apply')}}</PSButton>
+          <PSButton @click="hideModal" class="btn-lg" ghost data-dismiss="modal">{{trans('Cancel')}}</PSButton>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { mapGetters, mapActions } from 'vuex';
+  import PSButton from 'app/widgets/ps-button';
+  import CurrencyFormatForm from './CurrencyFormatForm';
+
+  export default {
+    name: 'currency-modal',
+    data() {
+      return {
+        showCustomizer: false
+      }
+    },
+    components: {
+      CurrencyFormatForm,
+      PSButton,
+    },
+    computed: {
+      ...mapGetters([
+        'editedLanguage',
+        'customData',
+        'showCurrencyModal'
+      ])
+    },
+    methods: {
+      ...mapActions([
+        'customizeCurrencyFormat',
+        'setShowCurrencyModal'
+      ]),
+      saveCustomCurrency() {
+        this.customizeCurrencyFormat();
+        this.hideModal();
+      },
+      showModal() {
+        $(this.$el).modal('show');
```

Comment:
:scream: Never use jQuery in vuejs, worst practice ever

Reply:
Really better to create a new one. We already have modal on our autoupgrade module => implement-vue pull request (as example)